tests/TestCase/Controller/Component/AuthComponentTest.php

```diff
@@ -23,6 +23,7 @@
 use Cake\Network\Request;
 use Cake\Network\Response;
 use Cake\ORM\TableRegistry;
+use Cake\Routing\Route\InflectedRoute;
```

Comment:
Yeah but if the tooling (default IDE settings) locally does it this way, it would be annoying if the cs tool did it the other way.